app/src/qvm-app-version.lisp

```diff
@@ -35,23 +35,36 @@
     :documentation "The git hash of the QVM repo.")
   )
 
-(defun latest-sdk-version ()
-  "Get the latest SDK quilc version, or NIL if unavailable."
+(defun latest-sdk-version (&key (proxy nil))
+  "Get the latest SDK qvm version, or NIL if unavailable."
   (handler-case
       (let* ((s (drakma:http-request "http://downloads.rigetti.com/qcs-sdk/version"
-                                     :want-stream t))
+                                     :want-stream t
+                                     :proxy proxy))
              (p (yason:parse s)))
         (multiple-value-bind (version success)
             (gethash "qvm" p)
           (when success
             version)))
-    (usocket:ns-host-not-found-error (condition)
+    (usocket:ns-error (condition)
       (declare (ignore condition))
+      (cl-syslog:rfc-log (*logger* :warning "Encountered a name resolution error when fetching latest SDK version. (~A)" condition)
+        (:msgid "LOG0001"))
+      nil)
+    (usocket:socket-error (condition)
+      (declare (ignore condition))
+      (cl-syslog:rfc-log (*logger* :warning "Encountered a socket error when fetching latest SDK version. (~A)" condition)
+        (:msgid "LOG0001"))
+      nil)
+    (sb-bsd-sockets:socket-error (condition)
```

Comment:
It's not clear to me why `usocket:socket-error` doesn't catch this on Windows. But it is what it is.